README.md

```diff
@@ -21,6 +21,7 @@ $ symfony-upgrade-fixer fix /path/to/code --dry-run
 
 | Name  | Description |
 | ----  | ----------- |
+| form_configure_options | The method AbstractType::setDefaultOptions(OptionsResolverInterface $resolver) have been renamed to AbstractType::configureOptions(OptionsResolver $resolver). |
 | form_events | The events PRE_BIND, BIND and POST_BIND were renamed to PRE_SUBMIT, SUBMIT and POST_SUBMIT. |
 | form_option_names | Options precision and virtual was renamed to scale and inherit_data. |
```

Comment:
You think grammar nazzi like me didn't notice. :)

https://github.com/symfony/symfony/blob/2.8/UPGRADE-3.0.md is full of inconsistency, I just followed.